content/influxdb/cloud-dedicated/process-data/visualize/chronograf.md

```diff
@@ -0,0 +1,125 @@
+
+## Create an InfluxDB connection
+
+1. In Chronograf, click **Configuration** in the left navigation bar,
+   and then click **{{< icon "plus" >}} Add Connection**.
+2. Enter your {{% cloud-name %}} connection credentials:
+
+    - **Connection URL:** {{% cloud-name %}} cluster URL
+
+      ```
+      https://cluster-id.influxdb.io
+      ```
+
+    - **Connection Name:** Name to uniquely identify this connection configuration
+    - **Username:** Arbitrary string _(ignored, but cannot be empty)_
+    - **Password:** InfluxDB [database token](/influxdb/cloud-dedicated/admin/tokens/)
+      with read permissions on the database you want to query
+    - **Telegraf Database Name:** InfluxDB [database](/influxdb/cloud-dedicated/admin/databases/)
+      Chronograf uses to populate parts of the application, including the Host List page (default is `telegraf`)
+    - **Default Retention Policy:** Default [retention policy](/influxdb/cloud-dedicated/reference/glossary/#retention-policy-rp)
+      _**(leave blank)**_
+
+        {{% note %}}
+#### DBRPs map to InfluxDB databases
+
+In {{% cloud-name %}}, databases and retention-policies (DBRPs) are no longer 
+separate entities in the data model. Rather than having one or more retention policies,
+An {{% cloud-name %}} database has a retention period, which defines the maximum
+age of data to retain in the database.
+InfluxQL queries still assume the 1.x DBRP convention, but with {{% cloud-name %}},
+InfluxQL queries are mapped to databases using the `database-name/retention-policy`
+naming convention. For example:
+
+```sql
+SELECT * FROM mydb.autogen.measurement
+```
+
+This query is routed to the {{% cloud-name %}} database with the name `mydb/autogen`.
+      {{% /note %}}
+
+3. Click **Add Connection**.
+4. Select the dashboards you would like to create, and then click **Next**.
+5. To configure a Kapacitor connection, provide the necessary credentials,
+   and then click **Continue**. Otherwise, click **Skip**.
+
+   <!-- <!-- _For information about using Kapacitor with InfluxDB Cloud or InfluxDB OSS {{< current-version >}}, -->
+   <!-- see [Use Kapacitor with InfluxDB](/influxdb/v2.7/tools/kapacitor/)._ -->
+
+6. Click **Finish**.
+
+## Query data in the Data Explorer
+
+1. In Chronograf, click **{{< icon "graph" "v2" >}} Explore** in the left navigation bar.
+2. Build and submit InfluxQL queries.
+
+{{% note %}}
+#### Schema information is not available
+
+{{% cloud-name %}} currently offers limited support of InfluxQL metaqueries, so
+schema information may not be available in the Data Explorer which limits
+the Data Explorer's query building functionality.
+This requires you to build your queries manually using
+[fully-qualified measurements](/influxdb/cloud-dedicated/reference/influxql/select/#fully-qualified-measurement)
+in the `FROM` clause. For example:
+
+```sql
+SELECT * FROM "db-name".."measurement-name"
```

Comment:
Is the `..` here because you omitted the dbrp? I was expecting `"db-name"."dbrp"."measurement-name"`. It might be nice to call out the `..` shorthand to just use the default dbrp.

Reply:
Good call. I'll add a bit more info.